tensornetwork/backends/jax/jax_backend_test.py

```diff
@@ -546,3 +545,22 @@ def fun(x, A, y):
   res1 = fun(x, A, y)
   res2 = fun_jit(x, A, y)
   np.testing.assert_allclose(res1, res2)
+
+
+def test_jit_args_args():
+  backend = jax_backend.JaxBackend()
+
+  def fun(x, A, y):
+    return jax.numpy.dot(x, jax.numpy.dot(A, y))
+
+  fun_jit = backend.jit(fun, static_argnums=(0,))
+  fun_jit_3 = backend.jit(fun, (0,))
```

Comment:
Wrong function to use keyword args on haha. 

do something like this.

```python

fun_jit = backend.jit(fun, (0,))
res1 = fun_jit(x, A, y)
res2 = fun_jit(x, A=A, y=y)
```

Reply:
ohh, OK got it